layers/sqlsrv/Dockerfile

```diff
@@ -3,32 +3,20 @@ FROM bref/build-php-$PHP_VERSION:1.1.4 AS ext
 
 ENV LD_LIBRARY_PATH=/usr/lib:/usr/lib64:$LD_LIBRARY_PATH
 
-RUN curl http://mirror.centos.org/centos/7/os/x86_64/Packages/unixODBC-2.3.1-14.el7.x86_64.rpm > /tmp/unixODBC-2.3.1-14.el7.x86_64.rpm
-RUN yum -y install /tmp/unixODBC-2.3.1-14.el7.x86_64.rpm
-RUN curl http://mirror.centos.org/centos/7/os/x86_64/Packages/unixODBC-devel-2.3.1-14.el7.x86_64.rpm > /tmp/unixODBC-devel-2.3.1-14.el7.x86_64.rpm
-RUN yum -y install /tmp/unixODBC-devel-2.3.1-14.el7.x86_64.rpm
+RUN yum -y install unixODBC-devel
```

Comment:
Could we specify a version here?

Reply:
I actually did have one specified, but then I noticed that `amzn-main` and `amzn2-core` contain different versions of that driver.
<img width="577" alt="Screen Shot 2021-02-28 at 12 50 52 PM" src="https://user-images.githubusercontent.com/6656197/109428130-9b899200-79c3-11eb-831c-604642a00a70.png">

Reply:
Oh, Okey. We ware not using amazonlinux:1 anymore, could we keep using 2.3.1-14?

Reply:
Sure, I reverted that change